src/mmw/js/src/modeling/modificationDescriptions.json

```diff
@@ -0,0 +1,73 @@
+{
+    chaparral: {
```

Comment:
All keys and values must be enclosed in double quotes in JSON files. http://json.org/